src/gui/torrentcontentmodel.cpp

```diff
@@ -275,10 +277,10 @@ QVector<int> TorrentContentModel::getFilePriorities() const
 
 bool TorrentContentModel::allFiltered() const
 {
-    for (const TorrentContentModelFile *fileItem : asConst(m_filesIndex))
-        if (fileItem->priority() != BitTorrent::FilePriority::Ignored)
-            return false;
-    return true;
+    return std::all_of(m_filesIndex.cbegin(), m_filesIndex.cend(), [](const TorrentContentModelFile *fileItem)
+    {
+        return (fileItem->priority() != BitTorrent::FilePriority::Ignored);
```

Comment:
I have trouble comprehending the logic here. Shouldn't `any_of` be used for this? or just `==`. But i assume it will `break` on true  with `any_of` so isn't that preferable?

Reply:
> or just ==

Oops... yes!